app/scripts/lib/app-start.js

```diff
@@ -264,18 +264,23 @@ function (
       var relier = this._relier;
       var screenInfo = new ScreenInfo(this._window);
       this._metrics = this._createMetrics(this._config.metricsSampleRate, {
-        lang: this._config.language,
-        service: relier.get('service'),
-        context: relier.get('context'),
-        entrypoint: relier.get('entrypoint'),
-        migration: relier.get('migration'),
+        able: this._able,
```

Comment:
@vladikoff and @zaach gave me a bit of a verbal assault for putting things in alphabetical order and making their review job more difficult. Back the alphabetical ordering changes out and do it in a separate PR.